### DISCUSSION TITLE
Icon Request - Text to Speech

### DISCUSSION THREAD

**Discussion by markabarrett** [FortAwesome/Font-Awesome#19749] [Icon Requests]
An icon that lets the user know it is possible to have the text on the screen read out loud to them

**Comment by sensibleworld** [comment]
Do have examples of what this icon would should look like?

**Comment by jrfbz** [comment]
A treatment like your CC closed caption icon would work, but with TTS.

**Comment by sensibleworld** [comment]
Worth noting that you can now build this icon in the new Icon Wizard: https://fontawesome.com/icons/wizard/sycGNHUjE1yMkhL5TUOSiO/share

<img width="1145" height="699" alt="CleanShot 2026-08-28 at 11 18 05@2x" src="https://github.com/user-attachments/assets/e6a0866b-6695-4a33-8d5e-e422681ed741" />
